ocs_ci/ocs/ui/base_ui.py

```diff
@@ -56,20 +56,32 @@ def __init__(self, driver: WebDriver):
         self.driver = driver
         self.screenshots_folder = os.path.join(
             os.path.expanduser(ocsci_config.RUN["log_dir"]),
-            f"screenshots_ui_{ocsci_config.RUN['run_id']}",
+            f"ui_logs_dir_{ocsci_config.RUN['run_id']}",
```

Comment:
I think you can define base_ui_logs_dir which will be 
```
base_ui_logs_dir = os.path.join(
            os.path.expanduser(ocsci_config.RUN["log_dir"]),
            f"ui_logs_dir_{ocsci_config.RUN['run_id']}",
)
            ```

Which you can just join with dom or screenshots_ui for following two folders variables. WDYT?

Reply:
done